src/main/java/de/uhd/ifi/se/decision/management/jira/extraction/view/macros/IssueMacro.java

```diff
@@ -45,9 +55,38 @@ public static String reformatCommentBody(String inputBody) {
 		}
 		return body;
 	}
-	
+
+	/**
+	 * Static function for other Macro Classes
```

Comment:
An alternative for integrating everything into the IssueMacro class would be to create an abstract superclass that the other macro classes inherit from/extend. This abstract ConDecMacro class should extend the BaseMacro class.